pycbc/events/single.py

```diff
@@ -303,7 +303,8 @@ def check(self, trigs, data_reader):
 
         # fill in a new candidate event
         candidate = {
-            f'foreground/{self.ifo}/{k}': cutall_trigs[k][i] for k in trigs
+            f'foreground/{self.ifo}/{k}': cut_trigs[k][sngl_idx][i]
```

Comment:
I'm a bit confused by the change to this line? Why do we need sngl_idx here now when we didn't before?

Reply:
`cutall_trigs` already had the `sngl_idx` included (line 285 above), but `cut_trigs` doesnt have that

Reply:
Ah I missed the change from `cutall_trigs` to `cut_trigs`